src/Core/Managed/Shared/OperationTelemetryExtensions.cs

```diff
@@ -40,7 +40,9 @@ public static void Start(this OperationTelemetry telemetry, long timestamp)
             // Stopwatch.GetTimestamp() is used (instead of ElapsedTicks) as it is thread-safe.
             telemetry.BeginTimeInTicks = timestamp;
 
+#if !CORE_PCL
```

Comment:
I just saw your IM. Thanks for fixing this. I didn't realize it had been broken for a while now. Yes, eventually we want RichPayloadEventSource in Core.PCL.